sources/packages/web/src/components/aest/OfferingChangeRequest.vue

```diff
@@ -13,7 +13,7 @@ import {
   OfferingDTO,
 } from "@/types";
 import { AESTRoutesConst } from "@/constants/routes/RouteConstants";
-import { BannerTypes } from "@/components/generic/Banner.models";
+import { BannerTypes } from "@/types/contracts/Banner";
```

Comment:
Thanks for moving the type. I believe that it can be included in the "@/types" index. In future reorganizations, it will cause less impact.